week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -89,12 +95,12 @@ function render() {
     });
 
     //add delete button to every row and render again
-    let delButton = document.createElement("button");
+    let delBut = document.createElement("button");//changed delButton todelBut 2
```

Comment:
I would probably have done this the other way around, `delButton` is a better name, but it gets the job done 👍